cypress/integration/edit_segment.spec.js

```diff
@@ -122,7 +115,10 @@ describe("Edit segment", () => {
             })
     })
 
-    it("should load content when clicking Revert", () => {
+    // The following test only works if the entries have a revision history
+    // Since the test-db is made when the tests are run, there is no revision history
+    // to test against
+    it.skip("should load content when clicking Revert", () => {
```

Comment:
Could you test the revert part after having cypress make a test edit?

Reply:
Likely, yes. Let me try

Reply:
Great suggestion! Putting the revert test after the "save on edit" test solved the problem.